packages/fiori/src/themes/Login.css

```diff
@@ -0,0 +1,103 @@
+* {
+    font-family: var(--sapFontFamily);
+}
+
+body {
+    margin: 0 auto;
+    background-color: var(--sapBackgroundColor);
+}
+
+:host {
+    width: 100%;
+    max-width: 650px;
+    display: flex;
+    margin-bottom: 25px;
+    -webkit-font-smoothing: antialiased;
+}
+
+.ui5-login {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    background: var(--sapTile_Background);
+    width: 100%;
+    border-radius: 2%;
+    -webkit-box-shadow: -1px 4px 15px -10px rgba(0, 0, 0, 0.45);
+    box-shadow: -1px 4px 15px -10px rgba(0, 0, 0, 0.45);
+    padding: 1rem 0;
+}
+
+.ui5-header-display {
+    width: 100%;
+    background-color: var(--sapButton_Emphasized_BorderColor);
+    border-radius: 5px 5px 0px 0px;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+}
+
+.fw-400 {

```

Comment:
We have CSS base vars for font-weight (--sapFont* something), we need to use them